<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title>Rudolf Schlechter</title> <meta name="author" content="Rudolf Schlechter"> <meta name="description" content="Personal website. Based on [*folio](https://github.com/bogoli/-folio) design. "> <meta name="keywords" content="jekyll, jekyll-theme, academic-website, portfolio-website"> <link href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha256-DF7Zhf293AJxJNTmh5zhoYYIMs2oXitRfBjY+9L//AY=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="https://unpkg.com/bootstrap-table@1.21.4/dist/bootstrap-table.min.css"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@6.4.0/css/all.min.css" integrity="sha256-HtsXJanqjKTc8vVQjO4YMhiqFoXkfBsjBWcX91T1jr8=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css" integrity="sha256-i1+4qU2G2860dGGIOJscdC30s9beBXjFfzjWLjBRsBg=" crossorigin="anonymous"> <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/github.css" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="/assets/img/bacteria.png"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://roschlec.github.io/"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/native.css" media="none" id="highlight_theme_dark"> <script src="/assets/js/theme.js?96d6b3e1c3604aca8b6134c7afdd5db6"></script> <script src="/assets/js/dark_mode.js?9b17307bb950ffa2e34be0227f53558f"></script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top"> <div class="container"> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item active"> <a class="nav-link" href="/">about<span class="sr-only">(current)</span></a> </li> <li class="nav-item "> <a class="nav-link" href="/blog/">blog</a> </li> <li class="nav-item "> <a class="nav-link" href="/publications/">publications</a> </li> <li class="nav-item "> <a class="nav-link" href="/projects/">projects</a> </li> <li class="nav-item "> <a class="nav-link" href="/portfolio/">portfolio</a> </li> <li class="nav-item "> <a class="nav-link" href="/cv/">curriculum vitae</a> </li> <li class="nav-item dropdown "> <a class="nav-link dropdown-toggle" href="#" id="navbarDropdown" role="button" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">submenus</a> <div class="dropdown-menu dropdown-menu-right" aria-labelledby="navbarDropdown"> <a class="dropdown-item" href="/publications/">publications</a> <div class="dropdown-divider"></div> <a class="dropdown-item" href="/projects/">projects</a> </div> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="fas fa-moon"></i> <i class="fas fa-sun"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5"> <div class="post"> <header class="post-header"> <h1 class="post-title"> <span class="font-weight-bold">Rudolf</span> Schlechter </h1> <p class="desc">Postdoc | Microbiology | Data Science | <a href="https://www.bcp.fu-berlin.de/en/biologie/arbeitsgruppen/mikrobiologie/ag_remus-emsermann/index.html" rel="external nofollow noopener" target="_blank">FU Berlin</a></p> </header> <article> <div class="profile float-right"> <figure> <picture> <img src="/assets/img/IMG_4668.png?949317ace3ace59713b4c65cf92bd4c1" class="img-fluid z-depth-1 rounded-circle" width="auto" height="auto" alt="IMG_4668.png" onerror="this.onerror=null; $('.responsive-img-srcset').remove();"> </picture> </figure> <div class="address"> <p>You can find me at the</p> <p><b>Institute of Microbiology</b></p> <p><b>Freie Universität Berlin</b></p> <p>Königin-Luise-Str. 12-16</p> <p>Berlin, Germany</p> </div> </div> <div class="clearfix"> <p>I'm a microbial ecologist based in Berlin. My research focuses on the ways microbes associate with each other and with plants. I'm currently interested in bacterial trait diversity and how it influences community assembly and ecosystem processes.</p> <p>I have worked in different research projects related to plant-pathogen interactions in grapes and <i>Arabidopsis thaliana</i>, spatial analysis of bacterial communities on the leaf surface, and generally working at the interface of microbiology, genomics, molecular biology and genetics, synthetic biology, and ecology.</p> <p>I'm a self-taught coder and passionate about using programming tools for data visualisation, and reproducible and open data analysis.</p> </div> <h2><a href="/news/" style="color: inherit;">news</a></h2> <div class="news"> <div class="table-responsive"> <table class="table table-sm table-borderless"> <tr> <th scope="row">Apr 17, 2024</th> <td> <a class="news-title" href="/news/announcement_3/">New paper accepted in Microbial Ecology!</a> </td> </tr> <tr> <th scope="row">Mar 4, 2024</th> <td> Attended my first Molecular Biology of Plants conference in Hennef. </td> </tr> </table> </div> </div> <h2><a href="/publications/" style="color: inherit;">selected publications</a></h2> <div class="publications"> <ol class="bibliography"> <li> <div class="row"> <div id="Schlechter2023-cm" class="col-sm-8"> <div class="title">Bacterial community complexity in the phyllosphere penalises specialists over generalists</div> <div class="author"> Rudolf O Schlechter, and Mitja N P Remus-Emsermann</div> <div class="periodical"> <em>bioRxiv</em>, Nov 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> </div> <div class="badges"> </div> <div class="abstract hidden"> <p>The leaf surface, i.e. the phylloplane, is an oligotrophic and heterogeneous environment due to its topography and uneven distribution of resources. Despite being a limiting environment, leaves host bacteria that are abundant and establish spatially-structured communities. However, factors that drive spatial distribution patterns are not well understood. Since leaf-associated bacteria can have beneficial effects to their host, understanding the rules of the community assembly can lead to novel strategies for crop protection. To investigate changes in population density and spatial distribution of bacteria in synthetic communities, we examined the behaviour of two prevalent bacterial groups in the Arabidopsis thaliana leaf microbiota: Methylobacterium spp. (specialists) and Sphingomonas spp. (generalists). We designed synthetic communities composed of two (S2) or three strains (S3) in a full factorial design and tested whether density and spatial structure of communities in S3 can be explained by pairwise comparisons in S2. Our results showed that specialists are more susceptible to changes in population densities and spatial distribution patterns than generalists, with lower densities and aggregation patterns when a specialist is in S3 than in S2. Additionally, pairwise comparisons were not sufficient to explain the observed patterns in S3, suggesting that higher-order interactions play a role in the resulting structure of complex communities at the micrometre scale. ### Competing Interest Statement The authors have declared no competing interest.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Schlechter2023-um" class="col-sm-8"> <div class="title">Metabolic resource overlap impacts competition among phyllosphere bacteria</div> <div class="author"> Rudolf O Schlechter, Evan J Kear, Michał Bernach, Daniela M Remus, and Mitja N P Remus-Emsermann</div> <div class="periodical"> <em>ISME J.</em>, Sep 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> </div> <div class="badges"> </div> <div class="abstract hidden"> <p>The phyllosphere is densely colonised by microbial communities, despite sparse and heterogeneously distributed resources. The limitation of resources is expected to drive bacterial competition resulting in exclusion or coexistence based on fitness differences and resource overlap between individual colonisers. We studied the impact of resource competition by determining the effects of different bacterial colonisers on the growth of the model epiphyte Pantoea eucalypti 299R (Pe299R). Resource overlap was predicted based on genome-scale metabolic modelling. By combining results of metabolic modelling and pairwise competitions in the Arabidopsis thaliana phyllosphere and in vitro, we found that ten resources sufficed to explain fitness of Pe299R. An effect of both resource overlap and phylogenetic relationships was found on competition outcomes in vitro as well as in the phyllosphere. However, effects of resource competition were much weaker in the phyllosphere when compared to in vitro experiments. When investigating growth dynamics and reproductive success at the single-cell resolution, resource overlap and phylogenetic relationships are only weakly correlated with epiphytic Pe299R reproductive success, indicating that the leaf’s spatial heterogeneity mitigates resource competition. Although the correlation is weak, the presence of competitors led to the development of Pe299R subpopulations that experienced different life histories and cell divisions. In some in planta competitions, Pe299R benefitted from the presence of epiphytes despite high resource overlap to the competitor strain suggesting other factors having stronger effects than resource competition. This study provides fundamental insights into how bacterial communities are shaped in heterogeneous environments and a framework to predict competition outcomes.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Schlechter2018-qq" class="col-sm-8"> <div class="title">Chromatic Bacteria - A Broad Host-Range Plasmid and Chromosomal Insertion Toolbox for Fluorescent Protein Expression in Bacteria</div> <div class="author"> Rudolf O Schlechter, Hyunwoo Jun, Michał Bernach, Simisola Oso, Erica Boyd, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Dian A Muñoz-Lintz, Renwick C J Dobson, Daniela M Remus, Mitja N P Remus-Emsermann' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '5'); ">4 more authors</span> </div> <div class="periodical"> <em>Front. Microbiol.</em>, Dec 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> </div> <div class="badges"> </div> <div class="abstract hidden"> <p>Differential fluorescent labeling of bacteria has become instrumental for many aspects of microbiological research, such as the study of biofilm formation, bacterial individuality, evolution, and bacterial behavior in complex environments. We designed a variety of plasmids, each bearing one of eight unique, constitutively expressed fluorescent protein genes in conjunction with one of four different antibiotic resistance combinations. The fluorophores mTagBFP2, mTurquoise2, sGFP2, mClover3, sYFP2, mOrange2, mScarlet-I, and mCardinal, encoding for blue, cyan, green, green-yellow, yellow, orange, red, and far-red fluorescent proteins, respectively, were combined with selectable markers conferring tetracycline, gentamicin, kanamycin, and/or chloramphenicol resistance. These constructs were cloned into three different plasmid backbones: a broad host-range plasmid, a Tn5 transposon delivery plasmid, and a Tn7 transposon delivery plasmid. The utility of the plasmids and transposons was tested in bacteria from the phyla Actinobacteria, Proteobacteria, and Bacteroidetes. We were able to tag representatives from the phylum Proteobacteria at least via our Tn5 transposon delivery system. The present study enables labeling bacteria with a set of plasmids available to the community. One potential application of fluorescently-tagged bacterial species is the study of bacteria-bacteria, bacteria-host, and bacteria-environment interactions.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Remus-Emsermann2018-sw" class="col-sm-8"> <div class="title">Phyllosphere microbiology: at the interface between microbial individuals and the plant host</div> <div class="author"> Mitja N P Remus-Emsermann, and Rudolf O Schlechter</div> <div class="periodical"> <em>New Phytol.</em>, Jun 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> </div> <div class="badges"> </div> <div class="abstract hidden"> <p>Contents Summary 1327 I. Introduction 1327 II. Individuality and the relevance of scales for the investigation of bacteria 1328 III. Bacterial aggregation and community patterning at the single-cell resolution 1329 IV. What are the effects on the plant host? 1330 V. Future directions and current questions 1331 Acknowledgements 1332 ORCID 1332 References 1332 Summary Leaf surfaces are home to diverse bacterial communities. Within these communities, every individual cell perceives its unique environment and responds accordingly. In this insight article, the perspective of the bacterial individual is assumed in an attempt to describe how the spatially heterogeneous leaf surface determines the fate of bacteria. To investigate behaviour at scales relevant to bacteria, single-cell approaches are essential. Single-cell studies provide important lessons about how current ?omics? approaches fail to give an accurate picture of the behaviour of bacterial populations in heterogeneous environments. Upcoming techniques will soon allow us to combine the power of single-cell and omics approaches.</p> </div> </div> </div> </li> </ol> </div> <div class="social"> <div class="contact-icons"> <a href="mailto:%72%6F%73%63%68%6C%65%63@%67%6D%61%69%6C.%63%6F%6D" title="email"><i class="fas fa-envelope"></i></a> <a href="https://orcid.org/0000-0002-5717-2917" title="ORCID" rel="external nofollow noopener" target="_blank"><i class="ai ai-orcid"></i></a> <a href="https://github.com/roschlec" title="GitHub" rel="external nofollow noopener" target="_blank"><i class="fab fa-github"></i></a> <a href="https://www.linkedin.com/in/roschlechter" title="LinkedIn" rel="external nofollow noopener" target="_blank"><i class="fab fa-linkedin"></i></a> <a href="https://twitter.com/rschlec" title="Twitter" rel="external nofollow noopener" target="_blank"><i class="fab fa-twitter"></i></a> </div> <div class="contact-note"> </div> </div> </article> </div> </div> <footer class="fixed-bottom"> <div class="container mt-0"> © Copyright 2024 Rudolf Schlechter. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. Photos from <a href="https://unsplash.com" target="_blank" rel="external nofollow noopener">Unsplash</a>. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.bundle.min.js" integrity="sha256-fgLAgv7fyCGopR/gBNq2iW3ZKIdqIcyshnUULC4vex8=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.8/dist/medium-zoom.min.js" integrity="sha256-7PhEpEWEW0XXQ0k6kQrPKwuoIomz8R8IYyuU1Qew4P8=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js"></script> <script defer src="https://unpkg.com/bootstrap-table@1.21.4/dist/bootstrap-table.min.js"></script> <script src="/assets/js/no_defer.js?d633890033921b33e0ceb13d22340a9c"></script> <script defer src="/assets/js/common.js?acdb9690d7641b2f8d40529018c71a01"></script> <script defer src="/assets/js/copy_code.js?c9d9dd48933de3831b3ee5ec9c209cac" type="text/javascript"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> </body> </html>