<!DOCTYPE html> <html lang="en"> <head> <meta http-equiv="Content-Type" content="text/html; charset=UTF-8"> <meta charset="utf-8"> <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=no"> <meta http-equiv="X-UA-Compatible" content="IE=edge"> <title>publications | Rudolf Schlechter</title> <meta name="author" content="Rudolf Schlechter"> <meta name="description" content="peer-reviewed publications, pre-prints and book chapters I co-authored."> <meta name="keywords" content="jekyll, jekyll-theme, academic-website, portfolio-website"> <link href="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha256-DF7Zhf293AJxJNTmh5zhoYYIMs2oXitRfBjY+9L//AY=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/css/mdb.min.css" integrity="sha256-jpjYvU3G3N6nrrBwXJoVEYI/0zw8htfFnhT9ljN3JJw=" crossorigin="anonymous"> <link defer rel="stylesheet" href="https://unpkg.com/bootstrap-table@1.21.4/dist/bootstrap-table.min.css"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free@6.4.0/css/all.min.css" integrity="sha256-HtsXJanqjKTc8vVQjO4YMhiqFoXkfBsjBWcX91T1jr8=" crossorigin="anonymous"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/academicons@1.9.1/css/academicons.min.css" integrity="sha256-i1+4qU2G2860dGGIOJscdC30s9beBXjFfzjWLjBRsBg=" crossorigin="anonymous"> <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700|Roboto+Slab:100,300,400,500,700|Material+Icons"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/github.css" media="" id="highlight_theme_light"> <link rel="shortcut icon" href="/assets/img/bacteria.png"> <link rel="stylesheet" href="/assets/css/main.css?d41d8cd98f00b204e9800998ecf8427e"> <link rel="canonical" href="https://roschlec.github.io/publications/"> <link rel="stylesheet" href="https://cdn.jsdelivr.net/gh/jwarby/jekyll-pygments-themes@master/native.css" media="none" id="highlight_theme_dark"> <script src="/assets/js/theme.js?96d6b3e1c3604aca8b6134c7afdd5db6"></script> <script src="/assets/js/dark_mode.js?9b17307bb950ffa2e34be0227f53558f"></script> </head> <body class="fixed-top-nav "> <header> <nav id="navbar" class="navbar navbar-light navbar-expand-sm fixed-top"> <div class="container"> <a class="navbar-brand title font-weight-lighter" href="/"><span class="font-weight-bold">Rudolf </span>Schlechter</a> <button class="navbar-toggler collapsed ml-auto" type="button" data-toggle="collapse" data-target="#navbarNav" aria-controls="navbarNav" aria-expanded="false" aria-label="Toggle navigation"> <span class="sr-only">Toggle navigation</span> <span class="icon-bar top-bar"></span> <span class="icon-bar middle-bar"></span> <span class="icon-bar bottom-bar"></span> </button> <div class="collapse navbar-collapse text-right" id="navbarNav"> <ul class="navbar-nav ml-auto flex-nowrap"> <li class="nav-item "> <a class="nav-link" href="/">about</a> </li> <li class="nav-item "> <a class="nav-link" href="/blog/">blog</a> </li> <li class="nav-item active"> <a class="nav-link" href="/publications/">publications<span class="sr-only">(current)</span></a> </li> <li class="nav-item "> <a class="nav-link" href="/projects/">projects</a> </li> <li class="nav-item "> <a class="nav-link" href="/portfolio/">portfolio</a> </li> <li class="nav-item "> <a class="nav-link" href="/cv/">curriculum vitae</a> </li> <li class="toggle-container"> <button id="light-toggle" title="Change theme"> <i class="fas fa-moon"></i> <i class="fas fa-sun"></i> </button> </li> </ul> </div> </div> </nav> <progress id="progress" value="0"> <div class="progress-container"> <span class="progress-bar"></span> </div> </progress> </header> <div class="container mt-5"> <div class="post"> <header class="post-header"> <h1 class="post-title">publications</h1> <p class="post-description">peer-reviewed publications, pre-prints and book chapters I co-authored.</p> </header> <article> <div class="publications"> <ol class="bibliography"> <li> <div class="row"> <div id="Kunzler2024-ns" class="col-sm-8"> <div class="title">Hitching a Ride in the Phyllosphere: Surfactant Production of Pseudomonas spp. Causes Co-swarming of Pantoea eucalypti 299R</div> <div class="author"> Michael Kunzler, Rudolf O Schlechter, Lukas Schreiber, and Mitja N P Remus-Emsermann</div> <div class="periodical"> <em>Microb. Ecol.</em>, Apr 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> </div> <div class="badges"> </div> <div class="abstract hidden"> <p>Here, we demonstrate the beneficial effect of surfactant-producing pseudomonads on Pantoea eucalypti 299R. We conducted a series of experiments in environments of increasing complexity. P. eucalypti 299R (Pe299R), and Pseudomonas sp. FF1 (Pff1) or Pe299R and surfactant-production deficient Pseudomonas sp. FF1::viscB (Pff1viscB) were co-inoculated in broth, on swarming agar plates, and on plants. In broth, there were no differences in the growth dynamics of Pe299R when growing in the presence of Pff1 or Pff1viscB. By contrast, on swarming agar plates, Pe299R was able to co-swarm with Pff1 which led to a significant increase in Pe299R biomass compared to Pe299R growing with Pff1viscB or in monoculture. Finally in planta, and using the single-cell bioreporter for reproductive success (CUSPER), we found a temporally distinct beneficial effect of Pff1 on co-inoculated Pe299R subpopulations that did not occur in the presence of Pff1viscB. We tested three additional surfactant-producing pseudomonads and their respective surfactant knockout mutants on PE299R on swarming agar showing similar results. This led us to propose a model for the positive effect of surfactant production during leaf colonization. Our results indicate that co-motility might be common during leaf colonization and adds yet another facet to the already manyfold roles of surfactants.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Kunzler2024-dv" class="col-sm-8"> <div class="title">Dataset: Surfactant production of Pseudomonas spp. causes co-swarming of Pantoea eucalypti 299R</div> <div class="author"> Michael Kunzler, Rudolf O Schlechter, and Mitja Remus-Emsermann</div> <div class="periodical"> <em>Zenodo</em>, Apr 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> </div> <div class="badges"> </div> <div class="abstract hidden"> <p>Raw and processed data include in vitro and in planta experiments of competition and swarming assays for Pantoea eucalypti 299R (Pe299R) and Pseudomonas sp. FF1 (PFF1) wild type or viscB mutant. In vitro: Plate reader - competition assay between Pe299R and PFF1 (wt or viscB mutant) Plate reader scan matrix - swarming assays In planta: CFU data CUSPER fluorescence data</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Schlechter2024-ah" class="col-sm-8"> <div class="title">Data set biofilm formation of antibiotic-resistant E. coli on lamb’s lettuce leaves</div> <div class="author"> Rudolf O Schlechter, Elisabet Marti, Mitja Remus-Emsermann, David Drissner, and Maria-Theresia Gekenidis</div> <div class="periodical"> <em>Zenodo</em>, Apr 2024 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> </div> <div class="badges"> </div> <div class="abstract hidden"> <p>This repository contains the metadata and raw data for the analysis of an in vitro screening of 174 antibiotic-resistance E. coli strains isolated from various sources to evaluate their ability and strength to form biofilms. This repository contains the raw data to characterise a subset of eleven E. coli strains in their population dynamics and persistance on lamb’s lettuce (Valerianella locusta) leaves. Raw images (czi format) of live/dead stain of selected strains in V. locusta leaves are provided. Data analysis and image processing scripts can be found in the GitHub repository associated to the manuscript.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Schlechter2023-cm" class="col-sm-8"> <div class="title">Bacterial community complexity in the phyllosphere penalises specialists over generalists</div> <div class="author"> Rudolf O Schlechter, and Mitja N P Remus-Emsermann</div> <div class="periodical"> <em>bioRxiv</em>, Nov 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> </div> <div class="badges"> </div> <div class="abstract hidden"> <p>The leaf surface, i.e. the phylloplane, is an oligotrophic and heterogeneous environment due to its topography and uneven distribution of resources. Despite being a limiting environment, leaves host bacteria that are abundant and establish spatially-structured communities. However, factors that drive spatial distribution patterns are not well understood. Since leaf-associated bacteria can have beneficial effects to their host, understanding the rules of the community assembly can lead to novel strategies for crop protection. To investigate changes in population density and spatial distribution of bacteria in synthetic communities, we examined the behaviour of two prevalent bacterial groups in the Arabidopsis thaliana leaf microbiota: Methylobacterium spp. (specialists) and Sphingomonas spp. (generalists). We designed synthetic communities composed of two (S2) or three strains (S3) in a full factorial design and tested whether density and spatial structure of communities in S3 can be explained by pairwise comparisons in S2. Our results showed that specialists are more susceptible to changes in population densities and spatial distribution patterns than generalists, with lower densities and aggregation patterns when a specialist is in S3 than in S2. Additionally, pairwise comparisons were not sufficient to explain the observed patterns in S3, suggesting that higher-order interactions play a role in the resulting structure of complex communities at the micrometre scale. ### Competing Interest Statement The authors have declared no competing interest.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Schlechter2023-um" class="col-sm-8"> <div class="title">Metabolic resource overlap impacts competition among phyllosphere bacteria</div> <div class="author"> Rudolf O Schlechter, Evan J Kear, Michał Bernach, Daniela M Remus, and Mitja N P Remus-Emsermann</div> <div class="periodical"> <em>ISME J.</em>, Sep 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> </div> <div class="badges"> </div> <div class="abstract hidden"> <p>The phyllosphere is densely colonised by microbial communities, despite sparse and heterogeneously distributed resources. The limitation of resources is expected to drive bacterial competition resulting in exclusion or coexistence based on fitness differences and resource overlap between individual colonisers. We studied the impact of resource competition by determining the effects of different bacterial colonisers on the growth of the model epiphyte Pantoea eucalypti 299R (Pe299R). Resource overlap was predicted based on genome-scale metabolic modelling. By combining results of metabolic modelling and pairwise competitions in the Arabidopsis thaliana phyllosphere and in vitro, we found that ten resources sufficed to explain fitness of Pe299R. An effect of both resource overlap and phylogenetic relationships was found on competition outcomes in vitro as well as in the phyllosphere. However, effects of resource competition were much weaker in the phyllosphere when compared to in vitro experiments. When investigating growth dynamics and reproductive success at the single-cell resolution, resource overlap and phylogenetic relationships are only weakly correlated with epiphytic Pe299R reproductive success, indicating that the leaf’s spatial heterogeneity mitigates resource competition. Although the correlation is weak, the presence of competitors led to the development of Pe299R subpopulations that experienced different life histories and cell divisions. In some in planta competitions, Pe299R benefitted from the presence of epiphytes despite high resource overlap to the competitor strain suggesting other factors having stronger effects than resource competition. This study provides fundamental insights into how bacterial communities are shaped in heterogeneous environments and a framework to predict competition outcomes.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Huerta2023-de" class="col-sm-8"> <div class="title">The WAK-like protein RFO1 acts as a sensor of the pectin methylation status in Arabidopsis cell walls to modulate root growth and defense</div> <div class="author"> Apolonio I Huerta, Gloria Sancho-Andrés, Juan Carlos Montesinos, Javier Silva-Navas, Solène Bassard, and <span class="more-authors" title="click to view 8 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '8 more authors' ? 'Corinne Pau-Roblot, Christopher Kesten, Rudolf O Schlechter, Susanne Dora, Temurkhan Ayupov, Jérôme Pelloux, Julia Santiago, Clara Sánchez-Rodrı́guez' : '8 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '5'); ">8 more authors</span> </div> <div class="periodical"> <em>Mol. Plant</em>, May 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> </div> <div class="badges"> </div> <div class="abstract hidden"> <p>Most organisms adjust their development according to the environmental conditions. For the majority, this implies the sensing of alterations to cell walls caused by different cues. Despite the relevance of this process, few molecular players involved in cell wall sensing are known and characterized. Here, we show that the wall-associated kinase-like protein RESISTANCE TO FUSARIUM OXYSPORUM 1 (RFO1) is required for plant growth and early defense against Fusarium oxysporum and functions by sensing changes in the pectin methylation levels in the cell wall. The RFO1 dwell time at the plasma membrane is affected by the pectin methylation status at the cell wall, regulating MITOGEN-ACTIVATED PROTEIN KINASE and gene expression. We show that the extracellular domain of RFO1 binds de-methylated pectin in vitro, whose distribution in the cell wall is altered during F. oxysporum infection. Further analyses also indicate that RFO1 is required for the BR-dependent plant growth alteration in response to inhibition of pectin de-methyl-esterase activity at the cell wall. Collectively, our work demonstrates that RFO1 is a sensor of the pectin methylation status that plays a unique dual role in plant growth and defense against vascular pathogens.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Schlechter2023-on" class="col-sm-8"> <div class="title">Microscopy images and datasets of Sphingomonas and Methylobacterium on Arabidopsis leaves</div> <div class="author"> Rudolf O Schlechter, Evan J Kear, and Mitja Remus-Emsermann</div> <div class="periodical"> <em>Zenodo</em>, May 2023 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> </div> <div class="badges"> </div> <div class="abstract hidden"> <p>This repository contains the raw data to analyse the population density at the CFU-level and single cell-resolution, and spatial distribution of bacterial communities composed of Methylobacterium and/or Sphingomonas species. Identities of each community can be found in metadata.csv and comm_id.csv. Data analysis can be found in the GitHub repository associated to the manuscript. File bacimg.tar.gz contains the pre-processed images of near-isogenic controls, two- and three-species communities (C, S2 and S3). Abbreviations: Fluorescence channels: C0: Channel 0 (Red fluorescence) C1: Channel 1 (Yellow/Cyan fluorescence) C2: Channel 2 (Cyan fluorescence) Independent experiments: e1: Experiment #1 e2: Experiment #2 Days post-inoculation: 7d: 7 days-post-inoculation 14d: 14 days-post-inoculation SynCom ID (SynID): c: near-isogenic controls (C) syn2: two-species communities (S2) syn3: three-species communities (S3) Community ID (ComID): com01-com15: Strain composition of each community (see comm_id.csv)</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Stocks2022-cg" class="col-sm-8"> <div class="title">Chromatic Bacteria v.2-A Himar1 Transposon-Based Delivery Vector to Extend the Host Range of a Toolbox to Fluorescently Tag Bacteria</div> <div class="author"> Christian Stocks, Rudolf O Schlechter, and Mitja N P Remus-Emsermann</div> <div class="periodical"> <em>Bacteria</em>, Feb 2022 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> </div> <div class="badges"> </div> <div class="abstract hidden"> <p>A recent publication described the construction and utility of a comprehensive “Chromatic Bacteria” toolbox containing a set of genetic tools that allows for fluorescently tagging a variety of Proteobacteria. In an effort to expand the range of bacteria taggable with the Chromatic Bacteria toolbox, a series of Himar1 transposon vectors was constructed to mediate insertion of fluorescent protein and antibiotic resistant genes. The Himar1 transposon was chosen as it is known to function in a wide range of bacterial species. To test the suitability of the new Himar1 Chromatic Bacteria plasmid derivatives, conjugations were attempted on recently isolated non-model organisms. Although we were unsuccessful in delivering the plasmids into Gram-positive bacterial isolates, we successfully modified previously recalcitrant isolates to the first set of the Chromatic Bacteria toolbox, such as Sphingomonas sp. Leaf357 and Acidovorax sp. Leaf84. This manuscript reports on the currently available plasmids and transposition success in different bacteria.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Schlechter2021-xd" class="col-sm-8"> <div class="title">Fluorescent Protein Expression as a Proxy for Bacterial Fitness in a High-Throughput Assay</div> <div class="author"> Rudolf O Schlechter, Evan J Kear, Daniela M Remus, and Mitja N P Remus-Emsermann</div> <div class="periodical"> <em>Appl. Environ. Microbiol.</em>, Aug 2021 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> </div> <div class="badges"> </div> <div class="abstract hidden"> <p>Bacterial growth is classically assessed by measuring the increases in optical density of pure cultures in shaken liquid media. Measuring growth using optical density has severe limitations when studying multistrain interactions, as it is not possible to measure the growth of individual strains within mixed cultures. Here, we demonstrated that constitutively expressed fluorescent proteins can be used to track the growth of individual strains in different liquid media. Fluorescence measurements were highly correlated with optical density measurements and cell counts. This allowed us to assess bacterial growth not only in pure cultures but also in mixed bacterial cultures and determine the impact of a competitor on a focal strain, thereby assessing relative fitness. Furthermore, we were able to track the growth of two different strains simultaneously by using fluorescent proteins with differential excitation and emission wavelengths. Bacterial densities measured by fluorescence yielded more consistent data between technical replicates than optical density measurements. Our setup employs fluorescence microplate readers that allow high throughput and replication. IMPORTANCE We expand on an important limitation of the concept of measuring bacterial growth, which is classically limited to one strain at a time. By adopting our approach, it is possible to measure the growth of several bacterial strains simultaneously with high temporal resolution and in a high-throughput manner. This is important to investigate bacterial interactions, such as competition and facilitation.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Miebach2020-gp" class="col-sm-8"> <div class="title">Litterbox-A gnotobiotic Zeolite-Clay System to Investigate Arabidopsis-Microbe Interactions</div> <div class="author"> Moritz Miebach, Rudolf O Schlechter, John Clemens, Paula E Jameson, and Mitja N P Remus-Emsermann</div> <div class="periodical"> <em>Microorganisms</em>, Mar 2020 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> </div> <div class="badges"> </div> <div class="abstract hidden"> <p>Plants are colonised by millions of microorganisms representing thousands of species withvarying effects on plant growth and health. The microbial communities found on plants arecompositionally consistent and their overall positive effect on the plant is well known. However,the effects of individual microbiota members on plant hosts and vice versa, as well as the underlyingmechanisms, remain largely unknown. Here, we describe “Litterbox”, a highly controlled system toinvestigate plant-microbe interactions. Plants were grown gnotobiotically, otherwise sterile, onzeolite-clay, a soil replacement that retains enough moisture to avoid subsequent watering.Litterbox-grown plants resemble greenhouse-grown plants more closely than agar-grown plantsand exhibit lower leaf epiphyte densities (106 cfu/g), reflecting natural conditions. Apolydimethylsiloxane (PDMS) sheet was used to cover the zeolite, significantly lowering thebacterial load in the zeolite and rhizosphere. This reduced the likelihood of potential systemicresponses in leaves induced by microbial rhizosphere colonisation. We present results of exampleexperiments studying the transcriptional responses of leaves to defined microbiota members andthe spatial distribution of bacteria on leaves. We anticipate that this versatile and affordable plantgrowth system will promote microbiota research and help in elucidating plant-microbe interactionsand their underlying mechanisms.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Schlechter2019-mz" class="col-sm-8"> <div class="title">Driving factors of epiphytic bacterial communities: A review</div> <div class="author"> Rudolf O Schlechter, Moritz Miebach, and Mitja N P Remus-Emsermann</div> <div class="periodical"> <em>J. Advert. Res.</em>, Sep 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> </div> <div class="badges"> </div> <div class="abstract hidden"> <p>Bacteria establish complex, compositionally consistent communities on healthy leaves. Ecological processes such as dispersal, diversification, ecological drift, and selection as well as leaf surface physicochemistry and topology impact community assembly. Since the leaf surface is an oligotrophic environment, species interactions such as competition and cooperation may be major contributors to shape community structure. Furthermore, the plant immune system impacts on microbial community composition, as plant cells respond to bacterial molecules and shape their responses according to the mixture of molecules present. Such tunability of the plant immune network likely enables the plant host to differentiate between pathogenic and non-pathogenic colonisers, avoiding costly immune responses to non-pathogenic colonisers. Plant immune responses are either systemically distributed or locally confined, which in turn affects the colonisation pattern of the associated microbiota. However, how each of these factors impacts the bacterial community is unclear. To better understand this impact, bacterial communities need to be studied at a micrometre resolution, which is the scale that is relevant to the members of the community. Here, current insights into the driving factors influencing the assembly of leaf surface-colonising bacterial communities are discussed, with a special focus on plant host immunity as an emerging factor contributing to bacterial leaf colonisation.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Jameson2019-vm" class="col-sm-8"> <div class="title">The Cytokinin Complex Associated With Rhodococcus fascians: Which Compounds Are Critical for Virulence?</div> <div class="author"> Paula E Jameson, Pragathi Dhandapani, Jiancheng Song, Marek Zatloukal, Miroslav Strnad, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Mitja N P Remus-Emsermann, Rudolf O Schlechter, Ondrej Novák' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '5'); ">3 more authors</span> </div> <div class="periodical"> <em>Front. Plant Sci.</em>, May 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> </div> <div class="badges"> </div> <div class="abstract hidden"> <p>Virulent strains of Rhodococcus fascians cause a range of disease symptoms, many of which can be mimicked by application of cytokinin. Both virulent and avirulent strains produce a complex of cytokinins, most of which can be derived from tRNA degradation. To test the three current hypotheses regarding the involvement of cytokinins as virulence determinants, we used PCR to detect specific genes, previously associated with a linear virulence plasmid, including two methyl transferase genes (mt1 and mt2) and fas4 (dimethyl transferase), of multiple strains of R. fascians. We inoculated Pisum sativum (pea) seeds with virulent and avirulent strains of R. fascians, monitored the plants over time and compared these to mock-inoculated controls. We used RT-qPCR to monitor the expression of mt1, mt2, and fas4 in inoculated tissues and LC-MS/MS to obtain a comprehensive picture of the cytokinin complement of inoculated cotyledons, roots and shoots over time. The presence and expression of mt1 and mt2 was associated with those strains of R. fascians classed as virulent, and not those classed as avirulent. Expression of mt1, mt2, and fas4 peaked at 9 days post-inoculation (dpi) in cotyledons and at 15 dpi in shoots and roots developed from seeds inoculated with virulent strain 602. Pea plants inoculated with virulent and avirulent strains of R. fascians both contained cytokinins likely to have been derived from tRNA turnover including the 2-methylthio cytokinins and cis-zeatin-derivatives. Along with the isopentenyladenine-type cytokinins, the levels of these compounds did not correlate with virulence. Only the novel 1- and 2-methylated isopentenyladenine cytokinins were uniquely associated with infection by the virulent strains and are, therefore, the likely causative factors of the disease symptoms.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Schlechter2019-oa" class="col-sm-8"> <div class="title">Delivering “Chromatic Bacteria” Fluorescent Protein Tags to Proteobacteria Using Conjugation</div> <div class="author"> Rudolf O Schlechter, and Mitja Np Remus-Emsermann</div> <div class="periodical"> <em>Bio Protoc</em>, Apr 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> </div> <div class="badges"> </div> <div class="abstract hidden"> <p>Recently, we published a large and versatile set of plasmids, the chromatic bacteria toolbox, to deliver eight different fluorescent protein genes and four combinations of antibiotic resistance genes to Gram-negative bacteria. Fluorescent tags are important tools for single-cell microbiology, synthetic community studies, biofilm, and host-microbe interaction studies. Using conjugation helper strain E. coli S17-1 as a donor, we show how plasmid conjugation can be used to deliver broad host range plasmids, Tn5 transposons delivery plasmids, and Tn7 transposon delivery plasmids into species belonging to the Proteobacteria. To that end, donor and recipient bacteria are grown under standard growth conditions before they are mixed and incubated under non-selective conditions. Then, transconjugants or exconjugant recipients are selected on selective media. Mutant colonies are screened using a combination of tools to ensure that the desired plasmids or transposons are present and that the colonies are not containing any surviving donors. Through conjugation, a wide range of Gram-negative bacteria can be modified without prior, often time-consuming, establishment of competent cell and electroporation procedures that need to be adjusted for every individual strain. The here presented protocol is not exclusive for the delivery of Chromatic bacteria plasmids and transposons, but can also be used to deliver other mobilizable plasmids to bacterial recipients.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Oso2019-wx" class="col-sm-8"> <div class="title">Utilisation of hydrocarbons and production of surfactants by bacteria isolated from plant leaf surfaces</div> <div class="author"> Simisola Oso, Matthew Walters, Rudolf O Schlechter, and Mitja N P Remus-Emsermann</div> <div class="periodical"> <em>FEMS Microbiol. Lett.</em>, Mar 2019 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> </div> <div class="badges"> </div> <div class="abstract hidden"> <p>Leaves are covered by a cuticle composed of long (C11-C20) and very-long chain hydrocarbons (&gt;C20), e.g. alkanes, fatty acids, alcohols, aldehydes, ketones and esters. In addition to these aliphatics, cyclic hydrocarbons may be present. Leaves are colonised by a variety of so-called epiphytic bacteria, which may have adapted to be able to utilise cuticle hydrocarbons. We tested the ability of a wide range of phylogenetically different epiphytic bacteria to utilise and grow on diesel and petroleum benzine and show that out of the 21 strains tested, nine had the ability to utilise diesel for growth. Only one strain was able to utilise petroleum benzine for growth. The ability to utilise hydrocarbons for growth correlated with the ability of the strains to produce surfactants and out of the 21 tested strains, 12 produced surfactants. Showing that 75% of the strains producing surfactants were able to degrade hydrocarbons. Our findings suggest that the ability to degrade hydrocarbons and to produce surfactants is highly prevalent in epiphytic bacteria. It is unclear if epiphytic bacteria utilise hydrocarbons originating from the cuticle of living leaves. The application of surfactant producing, hydrocarbon-utilising, epiphytic bacteria might serve as a method for hydrocarbon bioremediation.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Schlechter2018-qq" class="col-sm-8"> <div class="title">Chromatic Bacteria - A Broad Host-Range Plasmid and Chromosomal Insertion Toolbox for Fluorescent Protein Expression in Bacteria</div> <div class="author"> Rudolf O Schlechter, Hyunwoo Jun, Michał Bernach, Simisola Oso, Erica Boyd, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Dian A Muñoz-Lintz, Renwick C J Dobson, Daniela M Remus, Mitja N P Remus-Emsermann' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '5'); ">4 more authors</span> </div> <div class="periodical"> <em>Front. Microbiol.</em>, Dec 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> </div> <div class="badges"> </div> <div class="abstract hidden"> <p>Differential fluorescent labeling of bacteria has become instrumental for many aspects of microbiological research, such as the study of biofilm formation, bacterial individuality, evolution, and bacterial behavior in complex environments. We designed a variety of plasmids, each bearing one of eight unique, constitutively expressed fluorescent protein genes in conjunction with one of four different antibiotic resistance combinations. The fluorophores mTagBFP2, mTurquoise2, sGFP2, mClover3, sYFP2, mOrange2, mScarlet-I, and mCardinal, encoding for blue, cyan, green, green-yellow, yellow, orange, red, and far-red fluorescent proteins, respectively, were combined with selectable markers conferring tetracycline, gentamicin, kanamycin, and/or chloramphenicol resistance. These constructs were cloned into three different plasmid backbones: a broad host-range plasmid, a Tn5 transposon delivery plasmid, and a Tn7 transposon delivery plasmid. The utility of the plasmids and transposons was tested in bacteria from the phyla Actinobacteria, Proteobacteria, and Bacteroidetes. We were able to tag representatives from the phylum Proteobacteria at least via our Tn5 transposon delivery system. The present study enables labeling bacteria with a set of plasmids available to the community. One potential application of fluorescently-tagged bacterial species is the study of bacteria-bacteria, bacteria-host, and bacteria-environment interactions.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Remus-Emsermann2018-sw" class="col-sm-8"> <div class="title">Phyllosphere microbiology: at the interface between microbial individuals and the plant host</div> <div class="author"> Mitja N P Remus-Emsermann, and Rudolf O Schlechter</div> <div class="periodical"> <em>New Phytol.</em>, Jun 2018 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> </div> <div class="badges"> </div> <div class="abstract hidden"> <p>Contents Summary 1327 I. Introduction 1327 II. Individuality and the relevance of scales for the investigation of bacteria 1328 III. Bacterial aggregation and community patterning at the single-cell resolution 1329 IV. What are the effects on the plant host? 1330 V. Future directions and current questions 1331 Acknowledgements 1332 ORCID 1332 References 1332 Summary Leaf surfaces are home to diverse bacterial communities. Within these communities, every individual cell perceives its unique environment and responds accordingly. In this insight article, the perspective of the bacterial individual is assumed in an attempt to describe how the spatially heterogeneous leaf surface determines the fate of bacteria. To investigate behaviour at scales relevant to bacteria, single-cell approaches are essential. Single-cell studies provide important lessons about how current ?omics? approaches fail to give an accurate picture of the behaviour of bacterial populations in heterogeneous environments. Upcoming techniques will soon allow us to combine the power of single-cell and omics approaches.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Agurto2017-ks" class="col-sm-8"> <div class="title">RUN1 and REN1 Pyramiding in Grapevine (Vitis vinifera cv. Crimson Seedless) Displays an Improved Defense Response Leading to Enhanced Resistance to Powdery Mildew (Erysiphe necator)</div> <div class="author"> Mario Agurto*, Rudolf O Schlechter*, Grace Armijo, Esteban Solano, Carolina Serrano, and <span class="more-authors" title="click to view 3 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '3 more authors' ? 'Rodrigo A Contreras, Gustavo E Zúñiga, Patricio Arce-Johnson' : '3 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '5'); ">3 more authors</span> </div> <div class="periodical"> <em>Front. Plant Sci.</em>, May 2017 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> </div> <div class="badges"> </div> <div class="abstract hidden"> <p>Fungal pathogens are the cause of the most common diseases in grapevine and among them powdery mildew represents a major focus for disease management. Different strategies for introgression of resistance in grapevine are currently undertaken in breeding programs. For example, introgression of several resistance genes (R) from different sources for making it more durable and also strengthening the plant defense response. Taking this into account, we cross-pollinated P09-105/34, a grapevine plant carrying both RUN1 and REN1 pyramided loci of resistance to Erysiphe necator inherited from a pseudo-backcrossing scheme with Muscadinia rotundifolia and Vitis vinifera ’Dzhandzhal Kara,’ respectively, with the susceptible commercial table grape cv. ’Crimson Seedless.’ We developed RUN1REN1 resistant genotypes through conventional breeding and identified them by marker assisted selection. The characterization of defense response showed a highly effective defense mechanism against powdery mildew in these plants. Our results reveal that RUN1REN1 grapevine plants display a robust defense response against E. necator, leading to unsuccessful fungal establishment with low penetration rate and poor hypha development. This resistance mechanism includes reactive oxygen species production, callose accumulation, programmed cell death induction and mainly VvSTS36 and VvPEN1 gene activation. RUN1REN1 plants have a great potential as new table grape cultivars with durable complete resistance to E. necator, and are valuable germplasm to be included in grape breeding programs to continue pyramiding with other sources of resistance to grapevine diseases.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Wong2016-nj" class="col-sm-8"> <div class="title">A systems-oriented analysis of the grapevine R2R3-MYB transcription factor family uncovers new insights into the regulation of stilbene accumulation</div> <div class="author"> Darren Chern Jan Wong*, <em>Rudolf Schlechter*</em>, Alessandro Vannozzi, Janine Höll, Ibrahim Hmmam, and <span class="more-authors" title="click to view 4 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '4 more authors' ? 'Jochen Bogs, Giovanni Battista Tornielli, Simone Diego Castellarin, José Tomás Matus' : '4 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '5'); ">4 more authors</span> </div> <div class="periodical"> <em>DNA Res.</em>, Oct 2016 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> </div> <div class="badges"> </div> <div class="abstract hidden"> <p>R2R3-MYB transcription factors (TFs) belong to a large and functionally diverse protein superfamily in plants. In this study, we explore the evolution and function of this family in grapevine (Vitis vinifera L.), a high-value fruit crop. We identified and manually curated 134 genes using RNA-Seq data, and named them systematically according to the Super-Nomenclature Committee. We identified novel genes, splicing variants and grapevine/woody-specific duplicated subgroups, suggesting possible neo- and sub-functionalization events. Regulatory network analysis ascribed biological functions to uncharacterized genes and validated those of known genes (e.g. secondary cell wall biogenesis and flavonoid biosynthesis). A comprehensive analysis of different MYB binding motifs in the promoters of co-expressed genes predicted grape R2R3-MYB binding preferences and supported evidence for putative downstream targets. Enrichment of cis-regulatory motifs for diverse TFs reinforced the notion of transcriptional coordination and interaction between MYBs and other regulators. Analysis of the network of Subgroup 2 showed that the resveratrol-related VviMYB14 and VviMYB15 share common co-expressed STILBENE SYNTHASE genes with the uncharacterized VviMYB13. These regulators have distinct expression patterns within organs and in response to biotic and abiotic stresses, suggesting a pivotal role of VviMYB13 in regulating stilbene accumulation in vegetative tissues and under biotic stress conditions.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Armijo2016-jc" class="col-sm-8"> <div class="title">Grapevine Pathogenic Microorganisms: Understanding Infection Strategies and Host Response Scenarios</div> <div class="author"> Grace Armijo*, <em>Rudolf Schlechter*</em>, Mario Agurto, Daniela Muñoz, Constanza Nuñez, and <span class="more-authors" title="click to view 1 more author" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '1 more author' ? 'Patricio Arce-Johnson' : '1 more author'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '5'); ">1 more author</span> </div> <div class="periodical"> <em>Front. Plant Sci.</em>, Mar 2016 </div> <div class="periodical"> </div> <div class="links"> <a class="abstract btn btn-sm z-depth-0" role="button">Abstract</a> </div> <div class="badges"> </div> <div class="abstract hidden"> <p>Grapevine (Vitis vinifera L.) is one of the most important fruit crop worldwide. Commercial cultivars are greatly affected by a large number of pathogenic microorganisms that cause diseases during pre- and/or post-harvest periods, affecting production, processing and export, along with fruit quality. Among the potential threats, we can find bacteria, fungi, oomycete, or viruses with different life cycles, infection mechanisms and evasion strategies. While plant-pathogen interactions are cycles of resistance and susceptibility, resistance traits from natural resources are selected and may be used for breeding purposes and for a sustainable agriculture. In this context, here we summarize some of the most important diseases affecting V. vinifera together with their causal agents. The aim of this work is to bring a comprehensive review of the infection strategies deployed by significant types of pathogens while understanding the host response in both resistance and susceptibility scenarios. New approaches being used to uncover grapevine status during biotic stresses and scientific-based procedures needed to control plant diseases and crop protection are also addressed.</p> </div> </div> </div> </li> <li> <div class="row"> <div id="Cavallini2015-zc" class="col-sm-8"> <div class="title">The phenylpropanoid pathway is controlled at different branches by a set of R2R3-MYB C2 repressors in grapevine</div> <div class="author"> Erika Cavallini, José Tomás Matus, Laura Finezzo, Sara Zenoni, Rodrigo Loyola, and <span class="more-authors" title="click to view 5 more authors" onclick=" var element=$(this); element.attr('title', ''); var more_authors_text=element.text() == '5 more authors' ? 'Flavia Guzzo, Rudolf Schlechter, Agnès Ageorges, Patricio Arce-Johnson, Giovanni Battista Tornielli' : '5 more authors'; var cursorPosition=0; var textAdder=setInterval(function(){ element.text(more_authors_text.substring(0, cursorPosition + 1)); if (++cursorPosition == more_authors_text.length){ clearInterval(textAdder); } }, '5'); ">5 more authors</span> </div> <div class="periodical"> <em>Plant Physiol.</em>, Mar 2015 </div> <div class="periodical"> </div> <div class="links"> </div> <div class="badges"> </div> </div> </div> </li> </ol> </div> </article> </div> </div> <footer class="fixed-bottom"> <div class="container mt-0"> © Copyright 2024 Rudolf Schlechter. Powered by <a href="https://jekyllrb.com/" target="_blank" rel="external nofollow noopener">Jekyll</a> with <a href="https://github.com/alshedivat/al-folio" rel="external nofollow noopener" target="_blank">al-folio</a> theme. Hosted by <a href="https://pages.github.com/" target="_blank" rel="external nofollow noopener">GitHub Pages</a>. Photos from <a href="https://unsplash.com" target="_blank" rel="external nofollow noopener">Unsplash</a>. </div> </footer> <script src="https://cdn.jsdelivr.net/npm/jquery@3.6.0/dist/jquery.min.js" integrity="sha256-/xUj+3OJU5yExlq6GSYGSHk7tPXikynS7ogEvDej/m4=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.6.1/dist/js/bootstrap.bundle.min.js" integrity="sha256-fgLAgv7fyCGopR/gBNq2iW3ZKIdqIcyshnUULC4vex8=" crossorigin="anonymous"></script> <script src="https://cdn.jsdelivr.net/npm/mdbootstrap@4.20.0/js/mdb.min.js" integrity="sha256-NdbiivsvWt7VYCt6hYNT3h/th9vSTL4EDWeGs5SN3DA=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/masonry-layout@4.2.2/dist/masonry.pkgd.min.js" integrity="sha256-Nn1q/fx0H7SNLZMQ5Hw5JLaTRZp0yILA/FRexe19VdI=" crossorigin="anonymous"></script> <script defer src="https://cdn.jsdelivr.net/npm/imagesloaded@4/imagesloaded.pkgd.min.js"></script> <script defer src="/assets/js/masonry.js" type="text/javascript"></script> <script defer src="https://cdn.jsdelivr.net/npm/medium-zoom@1.0.8/dist/medium-zoom.min.js" integrity="sha256-7PhEpEWEW0XXQ0k6kQrPKwuoIomz8R8IYyuU1Qew4P8=" crossorigin="anonymous"></script> <script defer src="/assets/js/zoom.js"></script> <script defer src="https://unpkg.com/bootstrap-table@1.21.4/dist/bootstrap-table.min.js"></script> <script src="/assets/js/no_defer.js?d633890033921b33e0ceb13d22340a9c"></script> <script defer src="/assets/js/common.js?acdb9690d7641b2f8d40529018c71a01"></script> <script defer src="/assets/js/copy_code.js?c9d9dd48933de3831b3ee5ec9c209cac" type="text/javascript"></script> <script type="text/javascript">window.MathJax={tex:{tags:"ams"}};</script> <script defer type="text/javascript" id="MathJax-script" src="https://cdn.jsdelivr.net/npm/mathjax@3.2.0/es5/tex-mml-chtml.js"></script> <script defer src="https://polyfill.io/v3/polyfill.min.js?features=es6"></script> <script type="text/javascript">function progressBarSetup(){"max"in document.createElement("progress")?(initializeProgressElement(),$(document).on("scroll",function(){progressBar.attr({value:getCurrentScrollPosition()})}),$(window).on("resize",initializeProgressElement)):(resizeProgressBar(),$(document).on("scroll",resizeProgressBar),$(window).on("resize",resizeProgressBar))}function getCurrentScrollPosition(){return $(window).scrollTop()}function initializeProgressElement(){let e=$("#navbar").outerHeight(!0);$("body").css({"padding-top":e}),$("progress-container").css({"padding-top":e}),progressBar.css({top:e}),progressBar.attr({max:getDistanceToScroll(),value:getCurrentScrollPosition()})}function getDistanceToScroll(){return $(document).height()-$(window).height()}function resizeProgressBar(){progressBar.css({width:getWidthPercentage()+"%"})}function getWidthPercentage(){return getCurrentScrollPosition()/getDistanceToScroll()*100}const progressBar=$("#progress");window.onload=function(){setTimeout(progressBarSetup,50)};</script> </body> </html>